docker/ngc_overview_hugectr.md

```diff
@@ -0,0 +1,101 @@
+## What is Merlin for Recommender Systems?
+
+NVIDIA Merlin is a framework for accelerating the entire recommender systems pipeline on the GPU: from data ingestion and training to deployment. Merlin empowers data scientists, machine learning engineers, and researchers to build high-performing recommenders at scale. Merlin includes tools that democratize building deep learning recommenders by addressing common ETL, training, and inference challenges. Each stage of the Merlin pipeline is optimized to support hundreds of terabytes of data, all accessible through easy-to-use APIs. With Merlin, better predictions than traditional methods and increased click-through rates are within reach.
+
+The Merlin ecosystem has four main components: Merlin ETL, Merlin Dataloaders and Training, and Merlin Inference.
+
+## Merlin HugeCTR
+
+The merlin-hugectr container allows users to do preprocessing and feature engineering with NVTabular, and then train a deep-learning based recommender system model with HugeCTR, and serve the trained model on Triton Inference Server.
+
```

Comment:
After moving the preceding sentence to the preceding section, let's make the following into a bulleted list that identifies the key components of the container:

"The Merlin HugeCTR container includes the following key components to simplify developing and deploying your recommender system:"